CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ the detailed section referring to by linking pull requests or issues.
 
 - Implemented markdown for asset descriptions
 - Implemented list view for Broker UI
+- Asset Metadata is now editable
```

Comment:
Should we move the delete button to inside the edit dialog?
Deleting might not be a common use case and the button looks kind of lost there in the left corner :D

![image](https://github.com/sovity/edc-ui/assets/75735874/31ff9983-973b-42a5-9c22-5f63fb9ccbb5)

Reply:
moved to toolbar